### PR TITLE
Adds Stablehlo Batch Norm Training Op with Multiplier and Shift

### PR DESCRIPTION
**Pull request by pratham-mcw**
-Adds Stablehlo Batch Norm Training Implementation
 -Adds Stablehlo Batch Norm Training Unit Tests
 - `ComputeMean` and `ComputeVariance` functions have been moved to stablehlo_batch_norm_training.h in this commit

### CODE REVIEW

File: tensorflow/lite/core/api/flatbuffer_conversions.cc [hunk 1]
```diff
@@ -21,7 +21,7 @@ limitations under the License.
 #include <memory>
 
 #include "flatbuffers/flatbuffers.h"  // from @flatbuffers
-#include "flatbuffers/vector.h"  // from @flatbuffers
+#include "flatbuffers/vector.h"       // from @flatbuffers
```
Comment (swatheesh-mcw) [inline tensorflow/lite/core/api/flatbuffer_conversions.cc:24]: Can undo this change
